### DISCUSSION TITLE
Delete branch

### DISCUSSION THREAD

**Discussion by Jonny-T-cyber** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#857] [Q&A]

**Comment by im-coder-lg** [comment]
Oh no, you can't. That is a fork of the repo, and you made it when you hit the cute "Fork" button. However, to undo it, just delete the repo, and the fork will be erased from existence. Don't worry, this won't change anything. If you wanted to propose changes though, you need that fork and that `master` branch.

**Reply by sumeshir26** [reply]
In your repository click on settings in the top bar, scroll down and click delete.
![image](https://user-images.githubusercontent.com/68823982/214810861-987ec3a0-4467-4f4d-bace-89576f5daa4b.png)
![image](https://user-images.githubusercontent.com/68823982/214811093-d8a724d1-348d-4f0a-9b87-c19254ce4cb6.png)
